protocol/app/app.go

```diff
@@ -529,6 +537,21 @@ func New(
 		scopedIBCKeeper,
 	)
 
+	// Create ICA Host Keeper
+	app.ICAHostKeeper = icahostkeeper.NewKeeper(
+		appCodec,
+		keys[icahosttypes.StoreKey], // key
+		app.getSubspace(icahosttypes.SubModuleName), // paramSpace
+		app.IBCKeeper.ChannelKeeper,                 // ics4Wrapper, may be replaced with middleware such as ics29 fee
+		app.IBCKeeper.ChannelKeeper,                 // channelKeeper
+		&app.IBCKeeper.PortKeeper,                   // portKeeper
+		app.AccountKeeper,                           // accountKeeper
+		app.ScopedICAHostKeeper,                     // scopedKeeper
```

Comment:
Great callout - this was not initialized at this point.
To avoid similar confusion, I removed `ScopedXXXXKeeper` fields from the `app`. This pattern seems to come from example [implementation](https://github.com/search?q=%22make+scoped+keepers+public+for+test+purposes%22&type=code), but I believe these public scoped keeper fields are not utilized anywhere in our codebase.